pkg/v5/configmap/current.go

```diff
@@ -0,0 +1,42 @@
+package configmap
+
+import (
+	"context"
+
+	"github.com/giantswarm/microerror"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func (s *Service) GetCurrentState(ctx context.Context, config ConfigMapConfig) ([]*corev1.ConfigMap, error) {
+	var currentConfigMaps []*corev1.ConfigMap
+
+	guestK8sClient, err := s.guest.NewK8sClient(ctx, config.ClusterID, config.GuestAPIDomain)
+	if err != nil {
+		return nil, microerror.Mask(err)
+	}
+
+	// Namespaces used by all providers. Uses a map for deduping.
+	namespaces := map[string]bool{
+		metav1.NamespaceSystem: true,
+	}
```

Comment:
For now we just need to check `kube-system` for the Ingress Controller values. But the chartconfig CR includes the namespace of the config map so we need to support multiple namespaces.

More shared namespaces can be added here. The configmap resources can also add provider specific namespaces.